_emails/newsletter-2020-02.html

```diff
@@ -0,0 +1,108 @@
+{% capture text %}
+
+In our [continued](https://origami.ft.com/blog/2020/01/31/newsletter/) efforts
+towards openness we've created a [GitHub project
+board](https://github.com/orgs/Financial-Times/projects/83?fullscreen=true) that
+has a card for everything we're currently working on.
+
+When an issue is opened on any of our projects, it's added to the incoming
+column automatically by [a GitHub
+action](https://github.com/Financial-Times/origami-project-board-action/).
+
+We started with 348 things to do. We haven't yet seen that number drop below
+three hundred, but we've done 534 of them and we're working on the rest.
+
+![capture of Newman from Seinfeld with the caption "the mail never stops" ](https://www.ft.com/__origami/service/image/v2/images/raw/https://origami.ft.com/assets/images/2020-02-28-newsletter/the-mail-never-stops.png?source=origami)
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 2 -->
+{% include email/h3.html content="Layout post-mortem" %} {% capture text %}
+
+The latest version of Chrome made some unexpected changes to how they rendered
+things, and it broke the layout on a few of our internal projects. We reacted
+fast and got it fixed, and Lee wrote a great breakdown on how it happened and
+what we changed.
+
+Read it here: [CSS Grid in Chrome
+80](https://origami.ft.com/blog/2020/02/17/o-layout-chrome-80/)
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Thing 3 -->
+{% include email/h3.html content="Meter component" %}
+
+{% capture text %}
+
+We've created a new component that was in high demand, the
+[Meter](https://registry.origami.ft.com/components/o-meter).
+
+Similar components had been in use on internal and O&R products for a while, and
+now we've made this which helps increase consistency across all our teams and
+all their products.
+
+![the o-meter component in action](https://www.ft.com/__origami/service/image/v2/images/raw/https://origami.ft.com/assets/images/2020-02-28-newsletter/o-meter.png?source=origami)
+
+Isn't it beautiful?
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+<!-- Special thanks -->
+{% include email/h2.html content="Special Thanks" %}
+
+{% capture text %}
+
+Our thanks this month goes to Olga Averjanova for diving all the way in and
+creating a new component
+([o-meter](https://registry.origami.ft.com/components/o-meter)), and [a
+polyfill](https://github.com/Financial-Times/polyfill-library/pull/461)!
+
+A remarkable level of contribution for two weeks on the team! Thanks, Olga! 😊
+
+{% endcapture %}
+{% include email/markdown.html content=text %}
+
+
+<!-- Broader update -->
+{% include email/h2.html content="Broader Update" %}
+
+{% capture text %}A digest list of other things that have happened since our last update:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+{% capture text %}
+A digest list of other things that have happened this month:
```

Comment:
This was repeated, but also needs to be captured separately as `capture text` stops it from being unstyled serif text:
```suggestion
{% capture text %}A digest list of other things that have happened this month:{% endcapture %}
{% include email/markdown.html content=text %}

{% capture text %}
```